desktop-widgets/printer.cpp

```diff
@@ -14,7 +14,7 @@
 #include <QWebElement>
 #include "profile-widget/profilewidget2.h"
 
-Printer::Printer(QPaintDevice *paintDevice, const print_options &printOptions, const template_options &templateOptions, PrintMode printMode, bool inPlanner) :
+Printer::Printer(QPaintDevice *paintDevice, const print_options &printOptions, const template_options &templateOptions, PrintMode printMode, bool inPlanner, QWidget *parent = Q_NULLPTR) :
```

Comment:
Please don't reintroduce `Q_NULLPTR`. It never made sense. Use, `nullptr`, 0 or, if you insist on using a macro, `NULL`.